source/manuals/programming-languages/css.html.md.erb

```diff
@@ -17,15 +17,85 @@ owner_slack: '#frontend'
 
 ## Linting
 
-Your project should conform to GDS CSS/Sass linting rules, these are published as a [stylelint](https://stylelint.io) configuration: [stylint-config-gds](https://github.com/alphagov/stylelint-config-gds).
+Your project should conform to GDS CSS/Sass linting rules. These are published as a [stylelint][] configuration: [stylelint-config-gds][].
+
+Depending on your project needs, you may complement this initial set of rules with extra linting, for example:
+
+- other [stylelint plugins][]
+- [Prettier][] to enforce further code formatting conventions
+
+**Why**: Linting ensures consistency in the codebase and picks up on well-known issues. Using an shared set of rules across GDS ensures familiar conventions from one project to another.
+
+[stylelint]: https://stylelint.io
+[stylelint-config-gds]: https://github.com/alphagov/stylelint-config-gds
+[stylelint plugins]: https://stylelint.io/awesome-stylelint/#plugins
+[Prettier]: https://prettier.io/docs/en/cli.html
+
+### Tools
+
+#### Stylelint
+
+Stylelint is a widely used linter for CSS and CSS-like languages, like Sass. It focuses on checking code quality rather than code formatting. Its system of shareable configurations allows to share sets of rules to check your project against and plugins extends the rules it provides out of the box.
+
+`stylelint-config-gds` is a shareable config for Stylelint, providing the rules your code should follow, both for CSS and Sass files (each as separate configuration).
+
+In addition to this shared set of rules, you can use extra stylelint plugins or add rules as needs arise during the life of your project. In that area, automatically fixable rules are especially cheap to try out, as the tools will take care of updating your code for you.
+
+#### Prettier
+
+Prettier's only preoccupation is with [code formatting, not code quality][prettier-comparison]. 
+It can be used as a complement to ESLint for further automated formatting, with much more advanced decisions in terms of indentation, spaces, or line breaks.
+It runs as a separate command (`npx prettier`) and the [`eslint-config-prettier`][prettier-linters] ensures there'll be no conflicts between the rules of ESLint and the formatting of Prettier.
+
+[prettier-comparison]: https://prettier.io/docs/en/comparison
+[prettier-linters]: https://prettier.io/docs/en/integrating-with-linters
```

Comment:
There's a reference here to `eslint-config-linters`. I don't think there's a stylelint equivalent - [stylelint-config-prettier](https://github.com/prettier/stylelint-config-prettier) is archived since [Stylelint v15](https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules) deprecated all of the rules that conflicted with Prettier.

```suggestion
It runs as a separate command (`npx prettier`) and the should be no conflicts between the Stylelint rules and the formatting of Prettier.

[prettier-comparison]: https://prettier.io/docs/en/comparison
```